python/cudf/cudf/core/dataframe.py

```diff
@@ -111,6 +111,14 @@
 }
 
 
+def shape_mismatch_error(x, y):
```

Comment:
To go with Prem's suggested changes, we'd need to change this too:

```suggestion
def _shape_mismatch_error(x, y):
```